monitoring/statistics.h

```diff
@@ -30,16 +30,7 @@
 
 namespace rocksdb {
 
-enum TickersInternal : uint32_t {
```

Comment:
I think we should keep these internal ticker support, even though they are not being used. However we can make `INTERNAL_TICKER_ENUM_MAX` and `INTERNAL_HISTOGRAM_ENUM_MAX` visible in public header (include/rocksdb/statistics.h) so caller of `CreateDBStatistics` will know the start of custom ticker enum.